### PR TITLE
BAL-2261 Consider payment as successful when payment in Adyen has status pending

### PR DESCRIPTION
**Pull request by agneschristine**

### CODE REVIEW

**Review by jessevanmuijden — APPROVED** [review]

**Comment by tamasimrei** [comment]
Wait for Adyen to answer before we merge this.

File: src/Omnipay/Adyen/Message/CompletePurchaseRequest.php
Comment (cduran433) [inline src/Omnipay/Adyen/Message/CompletePurchaseRequest.php:156]: You might want to define the array in a constant for readable purpose.

**Review by cduran433 — APPROVED** [review]

**Pull request closed without merging**
